### DISCUSSION TITLE
WebAuthn with Windows Hello Supported?

### DISCUSSION THREAD

**Discussion by Junto026** [authelia/authelia#5585] [Troubleshooting] [answered]
I have Authelia using WebAuthn 2FA working very well with Mac/iPhone clients (it is pretty amazing and the user experience is great).

I tested from a Windows machine that has Windows Hello PIN setup.  On both Chrome and Edge browsers, I can get through the first factor authentication, but when I submit the second factor I get the message "You cancelled the assertion request."

My question is: is this supported?  If it should work I can dig deeper to troubleshoot.  If it's not supported I'll simply stop here :).

![Screen Shot 2023-06-19 at 12 44 37 PM](https://github.com/authelia/authelia/assets/88695895/9c104ad4-8baa-4022-9d96-1d2aee631dfa)

**Comment by james-d-elliott** [comment, marked as answer]
Platform authenticators are not yet supported. Will be something that is added most likely in 4.39, which will be much quicker than 4.38 is most likely.

**Comment by Junto026** [comment]
Got it, thank you!
